parse5-utils.js

```diff
@@ -36,8 +35,160 @@ function setAttribute(element, name, value) {
   }
 }
 
+function hasTagName(name) {
+  return function(node) {
+    return node.tagName === name;
+  };
+}
+
+function hasClass(name) {
+  return function(node) {
+    var attr = getAttribute(node, 'class');
+    if (!attr) {
+      return false;
+    }
+    return attr.split(' ').indexOf(name) > -1;
+  };
+}
+
+// OR an array of predicates
+function OR(rules) {
```

Comment:
My mistake, thought we were talking about `hasClass`. Using `arguments` is fine for me. I'll just make it an array internally until we have the spread operator.